src/Electron/electron.ts

```diff
@@ -157,10 +186,11 @@ export class ElectronContainer extends WebContainerBase {
     protected mainWindow: ElectronContainerWindow;
     protected electron: any;
     protected app: any;
-    protected browserWindow: any;
+    public browserWindow: any;
```

Comment:
I'm going to try adding Electron types in for better dev type checking on a future PR.  This would also cover using a real type for ipcMain and ipcRenderer instead of the base NodeJS.EventEmitter